src/doc/trpl/compound-data-types.md

```diff
@@ -228,8 +228,8 @@ enum Character {
 }
 ```
 
-An `enum` variant can be defined as most normal types. Below are some example
-types have been listed which also would be allowed in an `enum`.
+Most normal types are allowed as the variant components of an `enum`. Below are
+some examples.
```

Comment:
What about "Here are some examples:" which feels a bit more consistent with other places. I may just be remembering the frequency of stuff wrong though...